Rego/DefenderConfig.rego

```diff
@@ -443,6 +444,40 @@ SensitiveRules[{
     Policy := input.dlp_compliance_policies[_]
     Rules.ParentPolicyName == Policy.Name
     Policy.Enabled == true
+    Policy.Mode == "Enable"
+}
+
+SensitiveRules[{
+    "Name" : Rules.Name,
+    "ParentPolicyName" : Rules.ParentPolicyName,
+    "BlockAccess" : Rules.BlockAccess,
+    "BlockAccessScope" : Rules.BlockAccessScope,
+    "NotifyUser" : Rules.NotifyUser,
+    "NotifyUserType" : Rules.NotifyUserType,
+    "ContentNames" : ContentNames
+}] {
+    Rules := input.dlp_compliance_rules[_]
+    Rules.Disabled == false
+    Rules.IsAdvancedRule == true
+    Rules.AdvancedRule
+
+    Policy := input.dlp_compliance_policies[_]
+    Rules.ParentPolicyName == Policy.Name
+    Policy.Enabled == true
+    Policy.Mode == "Enable"
+
+    # Trim converted end-of-line "rn" delimters and convert single quotes to
+    # double for unmarshalling
+    RuleText := replace(replace(Rules.AdvancedRule, "rn", ""), "'", "\"")
+
+    ContentNames := regex.find_n(`(U.S. Social Security Number \(SSN\))|(U.S. Individual Taxpayer Identification Number \(ITIN\))|(Credit Card Number)`,
+                                 RuleText, -1)
+
+    Conditions := [ contains(RuleText, "U.S. Social Security Number (SSN)"),
+                    contains(RuleText, "U.S. Individual Taxpayer Identification Number (ITIN)"),
+                    contains(RuleText, "Credit Card Number")]
+
+    count([Condition | Condition = Conditions[_]; Condition == true]) > 0
```

Comment:
Update unification operator to assignment operator `count([Condition | Condition := Conditions[_]; Condition == true]) > 0`

```suggestion
    count([Condition | Condition := Conditions[_]; Condition == true]) > 0
```